readthedocs/doc_builder/python_environments.py

```diff
@@ -200,8 +200,8 @@ def is_obsolete(self):
         ])
 
     def _get_env_vars(self):
-        """Return env vars with their values of the project."""
-        env_vars = self.version.project.environmentvariable_set.values_list('name', 'value')
+        """Return env vars with their values of the project as a dictionary."""
+        env_vars = self.version.project.environment_variables
```

Comment:
I'd say that you can remove this private method completely and access the variable directly where it's used since it's used just once.

Reply:
Thank you.
I have pushed the changes.